setup.py

```diff
@@ -46,7 +46,7 @@
 AUTHOR_EMAIL        = "nansat-dev@googlegroups.com"
 PLATFORMS           = ["Linux", "OS X", "Windows"]
 ISRELEASED          = True
-VERSION             = os.getenv('NANSAT_RELEASE', '0.0.0dev').lstrip('v')
+VERSION             = os.getenv('NANSAT_RELEASE', '1.5.2').lstrip('v')
```

Comment:
Right, I need to find a better way than environment variables to get the version, it does not work with source installs.